src/app/client/src/app/modules/uci-admin/components/uci/uci.component.html

```diff
@@ -1,5 +1,5 @@
-<lib-lib-entry
+<!-- <lib-lib-entry
   [user]="userProfile" [baseUrl]="url" [blobUrl]="blobUrl" [botPhoneNumber]="uciBotPhoneNumber"
```

Comment:
@princegupta1131 
removed commented code if not required

Reply:
Removed the commeted line